6B/converterhelper.js

```diff
@@ -0,0 +1,68 @@
+/*
+    Tucker Kent
+    converterhelper.js
+    19SP_INFO_2134_WW Online - JavaScript II
+    Thoendel
+    15 April 2020
+*/
+
+class Converter {
+    //this is the base unit we want to convert to something else
+    constructor(unitToConvert) {
+        //we'll store this unit internally as _unitToConvert
+        let check = parseFloat(unitToConvert);
+        if(isNaN(check)) {
```

Comment:
here you are making sure that you read in the unit as a float....and check to see that it contains a number. So down below - with the twoDecimal method - you really just need to worry about formatting.